configurations/operator/perf-regression-latency-multitenant-2-clients.yaml

```diff
@@ -0,0 +1,34 @@
+test_duration: 540
+user_prefix: 'perf-regression-latency-2-clients'
+
+# NOTE: deploy 8 K8S nodes of the 'loader' type and then create 4 loader pods per DB cluster
+n_loaders: 8
+k8s_n_loader_pods_per_cluster: 4
+
+# NOTE: my calculated values for the 80-90% of load
+prepare_write_cmd: [
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..30000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=30000001..45000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=45000001..60000000",
+]
+# NOTE: 'stress_cmd_w', 'stress_cmd_r' and 'stress_cmd_m' are lists of strs where
+#       one str is dedicated for one of DB clusters.
+#       If it is just one str then the same command will be used for each of DB clusters
+# stress_cmd_r: [
```

Comment:
leave this as comment is confusing...
since the original throttle numbers here aren't as below
I think we can remove it (or at least put the one cluster number as reference here)